services/orders.js

```diff
@@ -1,8 +1,96 @@
 const moment = require('moment');
 const { Op } = require('sequelize');
 const { Order, OrderItem } = require('../models');
+const { take, countBy, meanBy, chain, sortBy } = require('lodash');
 
-function storeOrder({ grantAmount, orderReason, postcodeArea, items }) {
+function getAllOrders() {
+    return Order.findAll({
+        order: [['updatedAt', 'DESC']],
+        include: [
+            {
+                model: OrderItem,
+                as: 'items'
+            }
+        ]
+    }).then(orders => {
+        const normalisedDateFormat = 'YYYY-MM-DD';
+        // the date we added dropdown grant size options rather than free text
+        const LAUNCH_DATE = '2018-02-01';
+
+        let items = orders.reduce((acc, cur) => acc.concat(cur.items), []);
+
+        let totalOrders = orders.length;
+        let averageProductsPerOrder = Math.round(meanBy(orders, 'items.length'));
+        let averageItemQuantityPerOrder = Math.round(meanBy(orders, o => o.items.reduce((acc, cur) => acc + cur.quantity, 0)));
+
+        let orderByCount = arr => {
```

Comment:
I'd err towards breaking this out rather than relying on `chain`. Something like:

``` js
const mapped = map(arr, (count, code) => ({ code, count }));
return reverse(sortBy(mapped));
```